openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/Shared/Badge.less

```diff
@@ -1,17 +1,34 @@
+.announcement-badge-container {
+  position: absolute;
+  top: -11px;
+  left: 10px;
+  background: #fcf6fc;
+  border-radius: 4px;
+  border: 1px solid #dea4dc;
+  padding: 2px 8px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
 .announcement-badge {
-  width: 24px;
-  height: 24px;
-  background: #ffc143;
-  border: 1px solid #ffc143;
-  border-radius: 50%;
-  padding: 4px;
+  width: 12px;
+  height: 12px;
+  background: #fdfbfd;
```

Comment:
Can we use `less CSS` variables for colours?

Reply:
Updating the change.